server/centralized.go

```diff
@@ -155,6 +157,12 @@ func (c *Centralized) BatchRead(args *common.BatchReadRequest, reply *common.Bat
 	localArgs.ReplyChan = make(chan *common.BatchReadReply)
 	localArgs.Args = make([]common.PirArgs, len(args.Args))
 	for i, val := range args.Args {
+		//Handle pad requests.
+		if len(val.PirArgs) == 0 {
```

Comment:
What's a 'pad request'?
I'm trying to figure out why val.PirArgs would ever be an empty slice?